.travis.yml

```diff
@@ -3,8 +3,9 @@ dist: trusty
 sudo: required
 
 env:
-  - ci_docker=debian:buster-slim ci_distro=debian ci_suite=stretch
-  - ci_docker=debian:buster-slim ci_distro=debian ci_suite=stretch ci_configopts="--with-curl"
+  # debian has libsodium-dev, ubuntu doesn't in core at least
+  - ci_docker=debian:buster-slim ci_distro=debian ci_suite=stretch ci_pkgs="libsodium-dev"
```

Comment:
we have to explicitly provide `--with-libsodium` for both new tests.
Or change the autodetection to true in  `configure.ac`. For now we don't use the `libsodium` by default even if the package exists in the system:
```bash
$ dpkg -l libsodium-dev
ri  libsodium-dev:amd64 1.0.17-1     amd64        Network communication, cryptography and signaturing library - headers
```
```bash
$ ./configure  | grep sodium
    libsodium (ed25519 signatures):               no
```

Reply:
@cgwalters I'm not sure if you plan to enable the `libsodium` support by default, so not sure about the exact change.

And additional thing -- it seems to me that `ci_suite=stretch` should be `ci_suite=buster`, but again I'm unsure if it is the idea or issue. Sorry about that.

Reply:
Right, I had that originally and then got confused and thought it was enabled by default.  Fixed.